.gitignore

```diff
@@ -17,3 +17,4 @@ packages/
 *.tix
 /aula.yaml
 /state/AulaData/
+aula.cabal
```

Comment:
That won't work, because it will implicitly remove it from version control.  We want to have it version controlled, but not in the version you'd get.  I admit this is a bit strange.

I'll take care of it.